### ISSUE TITLE
Why TaskHandler.onEvent does not provide an argument with event info?

### ISSUE THREAD

**Issue by busslina** [Dev-hwang/flutter_foreground_task#154]
It only provides timestamp and send port. How do we know what event is?

**Comment by hoyo00** [comment]
what does event info mean?

the onEvent callback function is called according to the interval of ForegroundTaskOptions.

**Comment by busslina** [comment]
I mean to provide a description of the event. E.G. On Android you have some event functions like onCreate, onDestroy(), ... They provide that info in his name: event 'created', event 'destroyed'. But if you provide a generic 'onEvent' function you should specify the kind of that event in some way

**Comment by Dev-hwang** [comment]
Is it a naming problem? If so, what should I do? Any advice please

**Comment by Dev-hwang** [comment]
certainly, first-time users of this plugin will have a hard time understanding under what circumstances the onEvent callback function fires. thank you for your kind comments.
